### DISCUSSION TITLE
Persistent Volumes in a pod

### DISCUSSION THREAD

**Discussion by DinoMarlir** [jeremylvln/Shulker#497] [Q&A]
Hello, can you mount a persistent volume in a Minecraft server Pod?

I want to mount this volume to `/srv/shulker/volumes/myvolume`.

**Comment by jeremylvln** [comment]
Hi @DinoMarlir, I think this page from the documentation will help you: https://shulker.jeremylvln.fr/latest/guide/recipes/overriding-pod-properties.html#mounting-volumes

If it does not, do not hesitate to tell me, I've a lot of work to do to improve the documentation :)
